app/views/components/_toolbar_dropdown.html.erb

```diff
@@ -0,0 +1,21 @@
+<%
+  id ||= "toolbar-dropdown-#{SecureRandom.hex(4)}"
```

Comment:
Thanks for changing this - I don't think we actually use this id though?

Reply:
We're not, but we're doing this for most of our components. Happy to drop it

Reply:
It's not actually set on the details element so all we're doing is setting a variable.

Reply:
🤦‍♂️ now I get what you meant.